package.json

```diff
@@ -38,6 +38,7 @@
     "@mui/icons-material": "5.15.6",
     "@mui/material": "5.15.14",
     "@mui/material-nextjs": "5.15.11",
+    "@mui/x-data-grid": "^7.15.0",
```

Comment:
for security we recommend to lock de dependency versions

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/5/commits/169648a4487575eeb84eb6cb972c48cff4fa646d